db/migrate/20150606065713_add_sti_to_lookup_keys.rb

```diff
@@ -0,0 +1,27 @@
+class AddStiToLookupKeys < ActiveRecord::Migration
+  def up
+    add_column :lookup_keys, :type, :string
+    LookupKey.scoped.each do |key|
+      if key.is_param
+        key.type = 'PuppetclassLookupKey'
```

Comment:
Can this be done with an update_all based on is_param => false or true?  Saves iterating over a potentially large number of models.